net/nfdump/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nfdump
+PKG_VERSION:=1.6.18
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
+
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/phaag/nfdump/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=5d6046bf7faa34713b004c6cf8b3d30027c761c5ac22a3195d49388342e8147e
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+
+define Package/nfdump
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:= +libbz2
+  TITLE:= nfdump
+  URL:=https://github.com/phaag/nfdump/
+endef
+
+define Package/nfdump/description
+	NetFlow collecting and processing tools
+endef
+
+define Package/nfdump/conffiles
+/etc/config/nfcapd
+endef
+
+define Package/nfdump/install
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libnfdump-$(PKG_VERSION).so $(1)/usr/lib/
```

Comment:
Is the so missing a * at the end?

Reply:
This is the only library file the binaries seem to expect.

Reply:
Should be INSTALL_DATA then. .so files should not be executable.

Reply:
Now uses INSTALL_DATA.